### DISCUSSION TITLE
My app builds on Vercel but displayed a white page

### DISCUSSION THREAD

**Discussion by Bogzzz** [vercel/next.js#39240] [Help]
My app works fine on the local machine but I faced an issue when I pushed the app to Vercel. It build no problem except that it blink for a millisecond of what it should be and becomes white. In the console i see an error saying " Error rendering page:  TypeError: Cannot read properties of undefined (reading 'getInitialProps')" . I have spent a whole day researching what does this mean and found nothing that can help me. The only place where i use getInitialProps is in _documents, moved most of my requests to getServerSideProps in hope that it will help. But nothing changed. Please help .

**Comment by Babbili** [comment]
Hi @Bogzzz  is there a code to share of the page causing this issue, or a link to the repo.

**Reply by Bogzzz** [reply]
I wish i knew which part is causing it . Because the only place where i see the error is when i open the webpage on vercel . And all it says is that error. When i run it locally , i dont get any issues.@Babbili

**Reply by Bogzzz** [reply]
Solved it. I had Apollo for Links, and i guess Nextjs didnt like it. Deleted the files for links and it all works fine now.

**Reply by Babbili** [reply]
> Solved it. I had Apollo for Links, and i guess Nextjs didnt like it. Deleted the files for links and it all works fine now.

Great 😆 cheers
